### ISSUE TITLE
Disable navigation and change months dynamically

### ISSUE THREAD

**Issue by Chris0lsen** [arca-computing/MultipleDatePicker#59]
I had a use case recently where I wanted to compare the data from one month to that of another month, with two multiple-date-picker directives loading on the same page, without letting the user navigate both calendars individually, so I edited my copy of the directive to include a "disable-nav" attribute, and added the following code to the directive template:

`<div class="text-center picker-navigate picker-navigate-left-arrow" ng-class="{\'disabled\':disableBackButton}" ng-click="previousMonth()"><span ng-hide="disableNav">&lt;</span></div>`

After I did this, I saw that I couldn't change the month of the MDP, so I added a watch to override the nextMonth and previousMonth functions: 

`if(scope.disableNav){
                  scope.$watch('month', function(newMonth, old){
                    scope.generate();
                  }, true);
                }
`

And the monthChanged event on the 'master' MDP instance now updates the month on the 'slave' MSP instance, so when I navigate to a new month in one, it updates the other.

I don't know if this is too specific to be a PR, but it was an enhancement I needed, so I thought I'd offer it here for discussion!

**Comment by mgohin** [comment]
Hi,

thanks for the suggestion.
Couldn't you use `disallowBackPastMonths` and `disallowGoFuturMonths` to disable navigation ?

And you wanted the user to navigate one "master" calendar that changes a "slave" calendar too ? Not quite sure I understood the need you had but would be glad to understand :)

**Comment by mgohin** [comment]
Any news ?

**Comment by Chris0lsen** [comment]
Hi @mgohin,
Sorry I didn't follow up on this earlier this week. When I passed a month into the directive with `disallowBackPastMonths` and `disallowGoFuturMonths`, the user could still navigate to the _current month_ even if I had passed, for instance `{{month-1}}` to the directive - this is why I disabled the nav bar altogether.

The need for a master/slave setup was so while the user was inspecting data from the current month, he could compare it to the same data from the month prior.
